### PR TITLE
Slack Topic as Jira link 

### PR DESCRIPTION
**Pull request by gonchik**
Hi! 

This PR related to the issue  #1  

Cheers,
Gonchik Tsymzhitov

### CODE REVIEW

File: src/main/java/com/ullink/jira/slackit/Utils.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ private static String smartObfuscate(String key, String value) {
             return value;
         }
         //Second case, be strict on passwords
-        if ("password".equalsIgnoreCase("key")) {
+        if ("password".equalsIgnoreCase(key)) {
```
Comment (arnaudj) [inline src/main/java/com/ullink/jira/slackit/Utils.java]: :+1:

File: src/main/resources/js/slackit-users.js [hunk 2]
```diff
@@ -508,6 +530,18 @@ var SlackItClient = (function () {
 
     SlackItClient.prototype.onSlackItInvitePhaseInviteUser = function(self, user, channelId, channelName, nbGuests) {
         var _this = self;
+        var topic = AJS.params.baseURL + '/browse/' + _this.context.jiraIssueKey;
+        _this.rxClient.setTopicAsJiraLinkToChannelObservable(channelId, topic, _this.context.token, _this.context.slackApiBaseUrl).subscribe(
```
Comment (arnaudj) [inline src/main/resources/js/slackit-users.js]: minor suggestion: perhaps could be done when the channel is created initially, so any subsequent invite later (UI allows that) does not overwrite the channel topic (if modified by an user, e.g)
(onSlackItDialogDoStepCreateChannel?)

**Review by arnaudj — COMMENTED** [review]
lgtm so far

**Comment by arnaudj** [comment]
lgtm

**Comment by gonchik** [comment]
Thanks!

**Pull request merged**